### PR TITLE
Updated styles for pages: home, sign up, sign in, new game

### PR DESCRIPTION
**Pull request by kenwmak**
- Added image in home page
- Updated navbar
- Added footer
- Updated forms in New Game, Log in, Sign up to use simple form/bootstrap style

### CODE REVIEW

File: app/assets/stylesheets/master.css.scss [hunk 1]
```diff
@@ -1,21 +1,69 @@
...
-    text-align:left;
+  display: inline-block;
+  text-align:left;
+  font-family: helvetica, arial, sans-serif;
+  font-size: 18px;
+  list-style-type: none;
+}
+
+.hero {
+  background: url('home_image.jpg') no-repeat center center fixed;
```
Comment (lasley) [inline app/assets/stylesheets/master.css.scss]: Relative URIs will get you in trouble sometimes. `/home_image.jpg` (guessing at path) would be better, as it is not dependent on the current href

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -1,34 +1,21 @@
...
-  </div>
-
-  <div class="actions">
-    <%= f.submit "Sign up" %>
-  </div>
-<% end %>
-
-<%= render "devise/shared/links" %>
+<br />
+<br />
```
Comment (lasley) [inline app/views/devise/registrations/new.html.erb]: Are these just to push the div further down the screen? Might want to take a look at http://stackoverflow.com/questions/22196587/how-to-center-align-vertically-the-container-in-bootstrap

File: app/views/devise/sessions/new.html.erb [hunk 3]
```diff
@@ -1,26 +1,15 @@
...
-  <% end -%>
-
-  <div class="actions">
-    <%= f.submit "Log in" %>
-  </div>
-<% end %>
-
-<%= render "devise/shared/links" %>
+<br />
+<br />
```
Comment (lasley) [inline app/views/devise/sessions/new.html.erb]: Are these just to push the div further down the screen? Might want to take a look at http://stackoverflow.com/questions/22196587/how-to-center-align-vertically-the-container-in-bootstrap

File: app/views/games/index.html.erb [hunk 4]
```diff
@@ -1,17 +1,24 @@
-
-<div class="text-center">
-	<h1>Welcome!</h1>
-
-	<p>
-		Join an open game or select New Game to start your own
-	</p>
+<div class="hero">
+  <br />
+  <br />
```
Comment (lasley) [inline app/views/games/index.html.erb]: Are these just to push the div further down the screen? Might want to take a look at http://stackoverflow.com/questions/22196587/how-to-center-align-vertically-the-container-in-bootstrap

**Comment by kenwmak** [comment]
Made changes per feedback, ready to be pulled in.

**Pull request merged**